apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -5,6 +5,28 @@ import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.domain.builders.path
 import java.nio.file.Path
 
+/**
+ * Expects that the subject of the assertion (a [Path]) starts with the expected [Path].
```

Comment:
I tried to retrigger github action but without success at it seems. Maybe a new commit helps. Thus a small improvement (same can be done for the other kdoc)
```suggestion
 * Expects that the subject of the assertion (a [Path]) starts with the [expected] [Path].
```